src/components/admin/ArticleTableRows.js

```diff
@@ -47,11 +45,15 @@ class ArticleTableRows extends PureComponent {
           <Table.Cell>{formatDate(createdAt)}</Table.Cell>
           <Table.Cell>{formatDate(updatedAt)}</Table.Cell>
           <Table.Cell>
-            <Checkbox toggle defaultChecked={published} onChange={this.togglePublish(id)} />
+            <Checkbox
+              toggle
+              defaultChecked={published === 1}
+              onChange={(e, data) => this.togglePublish(article, data)}
```

Comment:
Not sure what's happening here, has id been split into article and data?

Reply:
When the value of the checkbox changes _(i.e. its clicked)_, this function is called with the event (`e`) and extra data (`data`). We pass the actual article and this extra data to the `togglePublish`.

`data` contains the new value of the checkbox.